### PR TITLE
feat: logger switch feature is added

### PR DESCRIPTION
**Pull request by chansun**
### What kind of change does this PR introduce? 
* [ ] bug fix
* [x] feature
* [ ] docs
* [ ] update

### What is the current behavior? 
HttpClient instance's logging feature for request/response is always on and cannot be turned off on the .spec file level.

### What is the new behavior?
HttpClient instance's logging feature now has a switch to turn on and off the logging feature for request/response.

### Other information
e.g., In a spec file, you may choose to turn off logging feature like this.
...
const httpClient = new HttpClient(HOST_URL, SERVICE_API_KEY, SERVICE_API_SECRET); // logging feature is on by default.
httpClient.logOff(); // turn off the logging feature and no more logs are printed
...

### CODE REVIEW

File: lib/logger-controller.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { Logger } from "tslog";
+
+export class LoggerController {
+  private logger: Logger;
+  private fakeLogger: FakeLogger = new FakeLogger();
+  private on: boolean = true;
+  constructor(logger: Logger) {
+    this.logger = logger;
+  }
+  public exec(): Logger | FakeLogger {
```
Comment (ryukato) [inline lib/logger-controller.ts]: Sorry I am not sure we really need this function to leave some logs, but I guess we can make `LoggerController` to a sort of wrapper for the `logger`. Please take a look at below.

```
class LoggerWrapper {
  private logger: Logger;
  constructor(logger: Logger) {
    this.logger = logger;
  }

  public debug(...args: unknown[]): void {
    if (this.isOn) {
      this.logger.debug(...);
    } else {
      // do nothing
    }
}

```

if we do like the above, then we don't need `FakeLogger`, don't we?

File: lib/logger-controller.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  }
+  error(...args: unknown[]): void {
+    args;
+    return
+  }
+  fatal(...args: unknown[]): void {
+    args;
+    return
+  }
+}
```
Comment (xymz) [inline lib/logger-controller.ts]: would you please install https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig if you using vscode, or install https://plugins.jetbrains.com/plugin/7294-editorconfig if you using intellij, or install https://github.com/editorconfig/editorconfig-vim if you using vim, or if you using emacs, install https://github.com/editorconfig/editorconfig-emacs or change to vim(was a joke).

Comment (xymz) [reply lib/logger-controller.ts]: reason : please insert newline each end of file

File: lib/logger-controller.ts (hunk 1 above)
Comment (chansun) [reply lib/logger-controller.ts]: The comment is reflected on : https://github.com/line/line-blockchain-developers-sdk-js/pull/86/commits/6de7cf0c660b7f4260c1c1456ec4eb2b22a899b3

File: lib/logger-wrapper.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  public warn(...args: unknown[]): void {
+    this.on ? this.logger.warn(...args) : null;
+  }
+  public error(...args: unknown[]): void {
+    this.on ? this.logger.error(...args) : null;
+  }
+  public fatal(...args: unknown[]): void {
+    this.on ? this.logger.fatal(...args) : null;
+  }
+}
```
Comment (xymz) [inline lib/logger-wrapper.ts]: please read https://github.com/line/line-blockchain-developers-sdk-js/pull/86#discussion_r1060395829

Comment (chansun) [reply lib/logger-wrapper.ts]: comment is reflected on : https://github.com/line/line-blockchain-developers-sdk-js/pull/86/commits/b824677bc2b8115d523a9bbd1d97f95fec80250c

File: lib/logger-controller.ts (hunk 2 above)
Comment (chansun) [reply lib/logger-controller.ts]: comment is reflected on : https://github.com/line/line-blockchain-developers-sdk-js/pull/86/commits/b824677bc2b8115d523a9bbd1d97f95fec80250c

**Review by ryukato — APPROVED** [review]

**Review by xymz — APPROVED** [review]

**Pull request merged**